src/components/volume/volume.js

```diff
@@ -288,7 +288,7 @@ class Volume extends Component {
           tabIndex="0"
           aria-label="Volume"
           className={style.controlButton}
-          onClick={() => this.onVolumeControlButtonClick()}
+          onMouseDown={() => this.onVolumeControlButtonMouseDown()}
```

Comment:
Let’s change all event handlers and omit the VolumeControl part, i.e. onMouseDoen, onKeyDown